### PR TITLE
fix(capture-sdk): Unknown qr code updated

### PR DESCRIPTION
**Pull request by AdnanYupi**
PIA-3279

Unknown QR code UI updated and logic of showing. 
To remove the flashing, between each show there is a one-second delay.

### CODE REVIEW

File: capture-sdk/sdk/src/main/java/net/gini/android/capture/camera/CameraFragmentImpl.java [hunk 1]
```diff
@@ -222,11 +224,21 @@ private void handleQRCodeDetected(@Nullable final PaymentQRCodeData paymentQRCod
...
-            mPaymentQRCodePopup.show(paymentQRCodeData);
-        }
+        if (mUnsupportedQRCodePopup.isShown() || mPaymentQRCodePopup.isShown())
+            return;
+
+        showQRCodeView(paymentQRCodeData);
+    }
+
+    private void showQRCodeView(PaymentQRCodeData data) {
+        new Handler(Looper.getMainLooper())
```
Comment (a-szotyori) [inline capture-sdk/sdk/src/main/java/net/gini/android/capture/camera/CameraFragmentImpl.java:234]: [Request] Two things I'm missing here to fulfill the requirements in the ticket ([see the first bullet point](https://ginis.atlassian.net/browse/PIA-3279)):
* The delay should only apply to the same QR code. If the QR code changes, then the delay should be ignored.
  When I tested there was a delay before showing the QR code UI for the new QR code.
* When detecting a QR code for the first time, then the delay should be ignored.
  When I tested there was a delay before showing the QR code UI for the first QR code.

**Review by a-szotyori — CHANGES_REQUESTED** [review]
Great progress! Some logic refinement and tablet layouts remaining.

File: capture-sdk/sdk/src/main/java/net/gini/android/capture/camera/CameraFragmentImpl.java (hunk 1 above)
Comment (AdnanYupi) [reply capture-sdk/sdk/src/main/java/net/gini/android/capture/camera/CameraFragmentImpl.java:234]: Ah shit I totally forgot tablets

**Review by a-szotyori — APPROVED** [review]
Thank you for the changes!

**Pull request merged**